common/lib/xmodule/xmodule/tests/test_capa_module.py

```diff
@@ -361,6 +361,108 @@ def test_showanswer_past_due(self):
                                             graceperiod=self.two_day_delta_str)
         self.assertFalse(still_in_grace.answer_available())
 
+    def test_showanswer_after_attempts_with_max(self):
+        '''
```

Comment:
Use double quotes for docstrings.

https://github.com/Stanford-Online/edx-platform/blob/master/openedx/stanford/style/python/docstrings.markdown